plantuml/LinkLoss/data_written.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+skinparam defaultTextAlignment center
+
+|App|
+
+|Link Loss Service|
+
+|Gap|
+start
+
+|Link Loss Service|
+:data written;
+-> write_request;
+
+:set alert level based on write request;
+-> level;
+
+if (level <= HIGH_ALERT ?) then (yes)
+    :set_alert_level(level)|
```

Comment:
I don't get it, who is "APP" in this context? do you mean service or user? I don't think we notify the user

Reply:
In this context, "App" refers to the user application, i.e. the program that instantiates a Link Loss Service.

I am not entirely sure what you mean by: "I don't think we notify the user". If you are referring to call to `set_alert_level`, please see line 106 of the implementation file: [LinkLossService](https://github.com/noonfom/mbed-os-experimental-ble-services/blob/ebf327d801610bb3261edd0521f43b7cdf0bc459/services/LinkLoss/source/LinkLossService.cpp#L106)

Reply:
how is the line 106 interacting with the user application?